src/processes.c

```diff
@@ -2235,6 +2246,8 @@ static int ps_read (void)
 	ps_submit_state ("blocked",  blocked);
 	ps_submit_state ("idle",     idle);
 	ps_submit_state ("wait",     wait);
+	ps_submit_state ("no_cmdline",  (count - num_cmdline));
```

Comment:
count was never defined in this if statement. I believe this would be a compile error against non-linux. Same below.

Reply:
See line 2129.

Reply:
Ahhh, they're defined in different blocks of variables over there. Alright, could we add the same comment to `count` as we have in other blocks as well? `/* returns number of processes */`

Reply:
Done.

Reply:
See my comment about naming.

Reply:
Done.